extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -1,92 +1,81 @@
 <div class="wds-global-navigation">
 	<div class="wds-global-navigation__content-bar">
-		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-logo">
+		<a href="<?= Sanitizer::encodeAttribute( $model['logo']['header']['href'] ); ?>" class="wds-global-navigation__logo">
 			<?= DesignSystemHelper::getSvg(
-				'wds-company-logo-fandom',
-				'wds-global-navigation__content-item-logo-fandom'
+				$model['logo']['header']['image'],
+				'wds-global-navigation__logo-fandom'
 			) ?>
-			<span class="wds-global-navigation__content-item-logo-powered-by">powered by</span>
+			<span class="wds-global-navigation__logo-powered-by">powered by</span>
 			<?= DesignSystemHelper::getSvg(
 				'wds-company-logo-wikia',
-				'wds-global-navigation__content-item-logo-wikia'
+				'wds-global-navigation__logo-wikia'
 			) ?>
 		</a>
-		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-is-games">Games</a>
-		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-is-movies">Movies</a>
-		<a href="#" class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-is-tv">TV</a>
-		<div class="wds-global-navigation__content-item wds-global-navigation__content-item-link wds-dropdown">
-			<div class="wds-dropdown-toggle wds-global-navigation__dropdown-toggle">
-				<span>Wikis</span>
-				<?= DesignSystemHelper::getSvg(
-					'wds-icons-dropdown-tiny',
-					'wds-icon wds-icon-tiny wds-dropdown-toggle-chevron'
-				) ?>
-			</div>
-			<div class="wds-dropdown-content wds-global-navigation__dropdown">
-				<ul class="wds-list">
-					<li><a href="#" class="wds-global-navigation__dropdown-link">Explore Wikis</a></li>
-					<li><a href="#" class="wds-global-navigation__dropdown-link">Community Central</a></li>
-				</ul>
-			</div>
-		</div>
-		<form class="wds-global-navigation__content-item wds-global-navigation__content-item-search">
-			<div class="wds-global-navigation__content-item-search-input-wrapper">
-				<label class="wds-global-navigation__content-item-search-label">
+		<?php
+		if ( isset( $model['verticals'] ) ):
+			foreach ( $model['verticals']['links'] as $link ):
+		?>
+		<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'linkBranded', [ 'model' => $link ] ); ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "model" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "DesignSystemGlobalNavigationService" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)